solidity/contracts/vortex/VortexBurner.sol

```diff
@@ -0,0 +1,529 @@
+
+    /**
+     * @dev returns the total amount of the burned governance tokens
+     *
+     * @return total amount of the burned governance tokens
+     */
+    function totalBurnedAmount() external view returns (uint256) {
+        return _totalBurnedAmount;
+    }
+
+    /**
+     * @dev converts the provided tokens to governance tokens and burns them
+     *
+     * @param tokens the tokens to convert
+     */
+    function burn(IERC20[] calldata tokens) external protected {
+        ITokenHolder feeWallet = networkFeeWallet();
+
+        // retrieve the burning strategy
+        Strategy memory strategy = burnStrategy(tokens, feeWallet);
+
+        // withdraw all token/ETH amounts to the contract
+        feeWallet.withdrawTokensMultiple(tokens, address(this), strategy.amounts);
+
+        // convert all amounts to the network token and record conversion amounts
+        IBancorNetwork network = bancorNetwork();
+
+        uint256[] memory networkTokenConversionAmounts = new uint256[](tokens.length);
+        uint256 grossNetworkTokenConversionAmount = 0;
+        uint256 totalGovTokenAmountToBurn = 0;
+
+        for (uint256 i = 0; i < strategy.paths.length; ++i) {
+            // avoid empty conversions
+            uint256 amount = strategy.amounts[i];
+            if (amount == 0) {
+                continue;
+            }
+
+            address[] memory path = strategy.paths[i];
+            IERC20 token = IERC20(path[0]);
+            uint256 value = 0;
+
+            if (token == _govToken) {
+                // if the source token is the governance token, don't try to convert it either, but rather include it in
+                // the amount to burn
+                //
+                // Please note that networkTokenConversionAmounts[i] will remain 0 in this case
+                totalGovTokenAmountToBurn = totalGovTokenAmountToBurn.add(amount);
+            } else if (token == _networkToken) {
+                // if the source token is the network token, don't try to convert it, but rather add its amount to the
+                // total amount to convert to the governance token
+                grossNetworkTokenConversionAmount = grossNetworkTokenConversionAmount.add(amount);
+
+                networkTokenConversionAmounts[i] = amount;
+            } else {
+                if (token != NATIVE_TOKEN_ADDRESS) {
+                    // if the source token is a regular token, approve the converter to withdraw the token amount
+                    ensureAllowance(token, network, amount);
+                } else {
+                    // if the source token is actually an ETH reserve, make sure to pass its value to the network
+                    value = amount;
+                }
+
+                // perform the actual conversion and optionally send ETH to the network
+                uint256 networkTokenConversionAmount =
+                    network.convertByPath{ value: value }(path, amount, 1, address(this), address(0), 0);
```

Comment:
You can use `convertByPath2` here (`convertByPath` is supposed to be deprecated at this point).

Reply:
convertByPath2 isn't deployed yet (and not scheduled to be deployed anytime soon).